### ISSUE TITLE
CIFAR-100 Training Issue: Difficulty Replicating Expected Results in CIFAR-100 Training

### ISSUE THREAD

**Issue by z3n0e** [YifeiZhou02/BT-2#1]
I attempted to reproduce the training process outlined in the paper for the baseline model. However, I consistently observed subpar performance compared to the reported results in the paper. Specifically, when applying the 'Independent' method, I achieved a Mean Average Precision (MAP) of 0.15 and a CMT top-1 accuracy of 0.25. These outcomes fall short of the corresponding figures documented in the paper.

This is my config:
arch_params:
  arch: ResNet50
  num_classes: 50 # This is the number of classes for architecture FC layer.
  embedding_dim: 128

optimizer_params:
  algorithm: adam
  lr: 0.001
  weight_decay: 0.000030517578125
  no_bn_decay: False
  momentum: 0.9
  nesterov: False

dataset_params:
  name: cifar100
  data_root: data_store/cifar-100-python # This should contain training and validation dirs.
  num_classes: 50 # This is the number of classes to include for training.
  num_workers: 16
  batch_size: 256

#not sure it is used
lr_policy_params:
  algorithm: cosine_lr
  warmup_length: 5
  epochs: 100
  lr: 0.001

seed: 42
epochs: 100
label_smoothing: 0.1
output_model_path: checkpoints/resnet50/cifar100_old.pt

**Comment by YifeiZhou02** [comment]
Hi, thanks for interest in our paper. We did notice that results on cifar100 are quite sensitive to slight differences of hyperparameters. Could you try using an sgd optimizer with learning rate 1.024 similar to used in this config (https://github.com/apple/ml-fct/blob/main/configs/cifar100_backbone_old.yaml)? Our training code for independent methods should be the same.

**Comment by YifeiZhou02** [comment]
You could also access our independent checkpoints from here (https://drive.google.com/drive/folders/1-Rl5DyneZqyc5o_zWT4Zikq4hPiKTimD?usp=sharing).

**Comment by z3n0e** [comment]
Thanks for the answer! It's a really interesting work.
